src/picknik_ur_gazebo_config/config/config.yaml

```diff
@@ -61,14 +61,17 @@ hardware:
       # ogre2 (which is the Gazebo default) is also an option if your system supports it.
       - gazebo_renderer: ${GAZEBO_RENDERER:-ogre}
       - kinematics_parameters_file:
+          # Load default_kinematics.yaml from ur_description/config/<ur_type>
           package: "ur_description"
-          path: "config"
+          path: "config/%>> hardware.type <<%/default_kinematics.yaml"
       - physical_parameters_file:
+          # Load physical_parameters.yaml from ur_description/config/<ur_type>
           package: "ur_description"
-          path: "config"
+          path: "config/%>> hardware.type <<%/physical_parameters.yaml"
       - visual_parameters_file:
+          # Load visual_parameters.yaml from ur_description/config/<ur_type>
           package: "ur_description"
-          path: "config"
+          path: "config/%>> hardware.type <<%/visual_parameters.yaml"
```

Comment:
I dislike our reliance on this meta-language that parses YAML substitutions, but I suppose this is actually a bug fix in that prior to this, we were using a single set of defaults?

But if this change is not needed, would love to avoid it.

Reply:
I am not a fan of either of the YAML substitutions. This logic can be moved to inside of the xacro, but I am also not a huge fan of of a lot of logic in xacro. I did it this way just because the base config already had it like this. I don't have a strong preference

Reply:
I see, yeah let's keep it then.